Documentation/StakeHolderRequirements.md

```diff
@@ -1,34 +1,34 @@
 # Stakeholder requirements
 
 ## Actuators:
-#### AT-1 The truck must drive forward and reverse
-#### AT-2 The truck must change direction while going forward, reverse or, being idle.  
+#### AT-1 The truck must be able to drive forward and in reverse
+#### AT-2 The truck must be able to change direction while going forward, reverse or, being idle.  
 #### AT-3 The truck must have right and left turn lights
 #### AT-9 The truck must have headlights
 #### AT-4 The truck must have lights on the roof
-#### AT-5 The truck may load and unload the semi Trailer
-#### AT-6 The truck must play audio signals
+#### AT-5 The truck must be able to load and unload the semi Trailer
+#### AT-6 The truck must have the ability to play audio signals
 #### AT-7 The truck must stop automatically if an obstacle is detected
 #### AT-8 The truck must stop automatically if the weight of the load exceeds 1 pound
 
 ## Communication:
-#### COM-1 The truck must communicate via Radiofrequency 
-#### COM-2 The truck must communicate via Bluetooth
-#### COM-3 The truck state must be communicated in a can bus
-#### COM-4 The truck application must be upgradeable via can bus
-#### COM-5 The truck must have a display on the front that shows a custom message 
+#### COM-1 The truck must communicate via Radiofrequency to a remote controller
+#### COM-2 The truck must communicate via Bluetooth to an app used for diagnostics and to update firmware
+#### COM-3 The truck state must be communicated via can bus through bluetooth
+#### COM-4 The truck application must be upgradeable via can bus (either through bluetooth or physical connection)
+#### COM-5 The truck must have a display on the front used to provide insight into the truck's status (excessive load, low battery, etc.)
 
 
 ## Sensors:
-#### SNS-1 The truck must take pictures or videos and store in a SD memory
-#### SNS-2 The truck must record audio and save it
+#### SNS-1 The truck must be able to take pictures or videos and store it in an SD memory
```

Comment:
I know it's a small detail, but I think this should be "The truck must be able to take pictures or videos and store them in an SD memory."

Everything else excellent job!